tests/test_utils_covariance.py

```diff
@@ -160,28 +160,39 @@ def test_covariances_cospectrum():
     assert_array_almost_equal(cosp_pr, cosp_sp, 6)
 
 
-def test_covariances_coherence():
+@pytest.mark.parametrize('coh', ['ordinary', 'instantaneous', 'lagged', 'imaginary', 'truc'])
+def test_covariances_coherence(coh):
```

Comment:
I would consider more ambitious tests eg that you actually test the statistical properties of the estimators eg that you have zero coh when you have a pure instantaneous mixing etc.

Reply:
ok, but it requires time. wip